src/applications/vaos/tests/list/va-appointments.unit.spec.jsx

```diff
@@ -14,180 +14,177 @@ const initialState = {
   },
 };
 
-describe('VAOS integration: appointment list future booked', () => {
-  describe('va appointments', () => {
-    it('should show information without facility details', async () => {
-      const appointment = getVAAppointmentMock();
-      appointment.attributes = {
-        ...appointment.attributes,
-        startDate: moment().format(),
-        clinicFriendlyName: 'C&P BEV AUDIO FTC1',
-        facilityId: '983',
-        sta6aid: '983GC',
-      };
-      appointment.attributes.vdsAppointments[0].currentStatus = 'FUTURE';
-      appointment.attributes.vdsAppointments[0].bookingNote =
-        'Some random note';
-
-      mockAppointmentInfo({ va: [appointment] });
-      const {
-        findByText,
-        baseElement,
-        getByText,
-        queryByText,
-      } = renderInReduxProvider(<FutureAppointmentsList />, {
-        initialState,
-        reducers,
-      });
-
-      const dateHeader = await findByText(
-        new RegExp(moment().format('dddd, MMMM D, YYYY'), 'i'),
-      );
-
-      expect(queryByText(/You don’t have any appointments/i)).not.to.exist;
-      expect(baseElement).to.contain.text('VA Appointment');
-      expect(baseElement).to.contain.text('Confirmed');
-      expect(baseElement).to.contain('.fa-check-circle');
-
-      expect(dateHeader).to.have.tagName('h3');
-      expect(getByText(/view facility information/i)).to.have.attribute(
-        'href',
-        '/find-locations/facility/vha_442GC',
-      );
-      expect(baseElement).not.to.contain.text('Some random note');
-      expect(getByText(/add to calendar/i)).to.have.tagName('a');
-      expect(getByText(/cancel appointment/i)).to.have.tagName('button');
+describe('VAOS integration: upcoming VA appointments', () => {
```

Comment:
All the changes in this file are indentation related, because I removed the `va appointments` block.